test/pom.xml

```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>com.mainstreethub</groupId>
     <artifactId>parent-pom</artifactId>
-    <version>1.5.1</version>
+    <version>1.6.6</version>
```

Comment:
Why 1.6.6 instead of 1.6.0?

Reply:
1.6.6 is the [latest release](https://github.com/mainstreethub/parent-pom/releases) that's now in Artifactory